slidez-fe-core/src/containers/user-menu/styles.module.scss

```diff
@@ -99,27 +105,33 @@
 }
 
 .userInfo {
-    display: flex;
-    flex-direction: row;
-    align-items: center;
-    justify-content: space-between;
-    margin: 14px;
-    font-weight: 400;
-    .avatar {
-        margin-right: 9px;
-        width: 32px !important;
-        height: 32px !important;
-        font-family: "Work Sans";
-    }
-    .userName {
-        font-size: 14px;
-    }
-    .userEmail {
-        padding-top: 3px;
-        font-size: 12px;
-        line-height: 14px;
-        color: rgba(0, 0, 0, 0.4);
-    }
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: space-between;
+  margin: 14px;
+  font-weight: 400;
+  .avatar {
+    margin-right: 9px;
```

Comment:
important fields from `.avatar` class above might be conflicting with this styles